scripts/pylint_extensions.py

```diff
@@ -134,6 +139,33 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+            #Executed if name is not having any default value in the function
+            #definition
+            else:
+                if name is None:
```

Comment:
L145-168 (almost the complete checker) are repetitions of the existing checker. This is highly unrecommended. Find an alternative and try to re-use stuff.

Reply:
Yes, I guess I could move the repeated lines to the outer for loop, and the `func_name` initialization to outside the for loop (as it's more general to the  entire class). What do you think?

Reply:
I think that can be done. Also, since we are now adding a new check to the same class, `ExplicitKeywordArgsChecker`, I suggest we rename the class too, to something more general -- may be `KeywordArgsChecker`.

Reply:
Done! PTAL